d3ploy/demand_driven_deployment_inst.py

```diff
@@ -200,83 +202,114 @@ def enter_notify(self):
             self.buffer_type_dict = di.build_buffer_type_dict(
-                self.buffer_type, self.commod_list)
+                self.buffer_type, self.commod_list
+            )
             for commod in self.commod_list:
-                lib.TIME_SERIES_LISTENERS["supply" +
-                                          commod].append(self.extract_supply)
-                lib.TIME_SERIES_LISTENERS["demand" +
-                                          commod].append(self.extract_demand)
+                lib.TIME_SERIES_LISTENERS["supply" + commod].append(
+                    self.extract_supply
+                )
+                lib.TIME_SERIES_LISTENERS["demand" + commod].append(
+                    self.extract_demand
+                )
                 self.commodity_supply[commod] = defaultdict(float)
                 self.commodity_demand[commod] = defaultdict(float)
             for child in self.children:
                 itscommod = self.fac_commod[child.prototype]
-                self.installed_capacity[itscommod][0] += self.commodity_dict[itscommod][child.prototype]['cap']
+                self.installed_capacity[itscommod][0] += self.commodity_dict[
+                    itscommod
+                ][child.prototype]["cap"]
             self.fresh = False
 
     def decision(self):
         """
-        This is the tock method for decision the institution. Here the institution determines the difference
-        in supply and demand and makes the the decision to deploy facilities or not.
+        This is the tock method for decision the institution. Here the
+        institution determines the difference in supply and demand and
+        makes the the decision to deploy facilities or not.
         """
         time = self.context.time
         for commod, proto_dict in self.commodity_dict.items():
             diff, supply, demand = self.calc_diff(commod, time)
-            lib.record_time_series('calc_supply' + commod, self, supply)
-            lib.record_time_series('calc_demand' + commod, self, demand)
+            lib.record_time_series("calc_supply" + commod, self, supply)
+            lib.record_time_series("calc_demand" + commod, self, demand)
             if diff < 0:
                 if self.installed_cap:
                     deploy_dict = solver.deploy_solver(
-                        self.installed_capacity, self.commodity_dict, commod, diff, time)
+                        self.installed_capacity,
+                        self.commodity_dict,
+                        commod,
+                        diff,
+                        time,
+                    )
                 else:
                     deploy_dict = solver.deploy_solver(
-                        self.commodity_supply, self.commodity_dict, commod, diff, time)
+                        self.commodity_supply,
+                        self.commodity_dict,
+                        commod,
+                        diff,
+                        time,
```

Comment:
there should definitely not be a comma here. there's no way this doesn't throw an error.

Reply:
I reverted all pep8 changes